prefixdb_test.go

```diff
@@ -49,7 +49,63 @@ func TestWithGolevelDB(t *testing.T) {
 
 	pdb := NewPrefixDB(db, bz("key"))
 
-	t.Run("PrefixDB", func(t *testing.T) { Run(t, pdb) })
+	t.Run("GolevelDB", func(t *testing.T) { Run(t, pdb) })
+}
+
+func TestWithClevelDB(t *testing.T) {
+	name := fmt.Sprintf("test_%x", randStr(12))
+	defer cleanupDBDir("", name)
```

Comment:
We can lean on the test harness to create and clean up test directories for us:

```suggestion
	dir := t.TempDir()
	path := filepath.Join(dir, "cleveldb")
	db, err := NewCLevelDB(path, "")
	require.NoError(t, err)
```

The test will auto-cleanup the directory. We might want to be sure it's closed first, though:

```go
defer func() { assert.NoError(t, db.Close()) }()
```

or words to that effect.